CoreFoundation/String.subproj/CFAttributedString.c

```diff
@@ -17,7 +17,7 @@
 #include "CFRuntime_Internal.h"
 
 #if (TARGET_OS_MAC || TARGET_OS_WIN32) && DEPLOYMENT_RUNTIME_OBJC
-#import <Foundation/NSAttributedString.h>
+#include <Foundation/NSAttributedString.h>
```

Comment:
This should be import

Reply:
The `TARGET_OS_WIN32` means that we will come down this path, which does mean that even clang will see that and object.

Reply:
DEPLOYMENT_RUNTIME_OBJC means I wont for swift-corelibs-foundation.

Reply:
Ah, fair enough.  I'll revert this bit to an import then.